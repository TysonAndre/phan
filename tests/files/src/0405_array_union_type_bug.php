<?php

class Common{
        const TYPE_INT = 'i';
}

class CC extends Common {
    protected static $mappings = [
        'key' => ['name' => 'name', 'type' => self::TYPE_INT],
    ];
    protected static $other_mappings = [
        'key' => ['name' => 'name', 'type' => self::TYPE_MISSING],
    ];
}

