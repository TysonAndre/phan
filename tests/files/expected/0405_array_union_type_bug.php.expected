%s:12 PhanUndeclaredConstantOfClass Reference to undeclared class constant \CC::TYPE_MISSING
